.github/workflows/build-test-verilator.yml

```diff
@@ -30,7 +30,7 @@ jobs:
     steps:
       - uses: actions/checkout@v3
         with:
-          submodules: 'true'
```

Comment:
I think this one should still checkout all submodules since it spcifically depends on verilator